src/components/box.js

```diff
@@ -63,9 +67,7 @@
         onMouseEnter={handleMouseEnter}
         onMouseLeave={handleMouseLeave}
         style={{
-          backgroundImage: interactionBackground
-            ? `url("${interactionBackground}")`
-            : 'auto',
+          backgroundImage: interactionBackground || hasBackgroundImage,
```

Comment:
Here the backgroundImage is either a string or a boolean which is also a bit inconsistent. maybe find a way to check the desired states but provide only one type of value to the prop itself to avoid polymorphism down the line

Reply:
I've refactored this a bit. Now when an interaction is used it uses the inline styling to apply it, otherwise, no inline styling is applied, and the optional background is handled by the `styles` section. This also cleans up the types.